src/PowerShell/Microsoft.WinGet.Client.Engine/Commands/CliCommand.cs

```diff
@@ -70,17 +70,33 @@ public void GetSettings(bool asPlainText)
         /// <param name="name">Name of source.</param>
         /// <param name="arg">Arg of source.</param>
         /// <param name="type">Type of source.</param>
-        public void AddSource(string name, string arg, string type)
+        /// <param name="trustLevel">Trust level of source.</param>
+        /// <param name="isExplicit">Make source explicit.</param>
+        public void AddSource(string name, string arg, string type, string trustLevel, bool isExplicit)
         {
             Utilities.VerifyAdmin();
+            string parameters = string.Empty;
+
             if (string.IsNullOrEmpty(type))
             {
-                _ = this.Run("source", $"add --name {name} --arg {arg}", 300000);
+                parameters = $"add --name {name} --arg {arg}";

```

Comment:
Not related to this change, but this code is not resilient to spaces in the arguments.